### PR TITLE
adding Status to functions of PaddedBytes

### PR DESCRIPTION
**Pull request by mo271**

### CODE REVIEW

File: lib/jxl/icc_codec.cc [hunk 1]
```diff
@@ -181,33 +181,33 @@ Status UnpredictICC(const uint8_t* enc, size_t size, PaddedBytes* result) {
...
 
       if (tagcode == kCommandTagXYZ) {
         JXL_RETURN_IF_ERROR(CheckIs32Bit(tagstart + tagsize * 2));
-        AppendKeyword(kGxyzTag, result);
-        AppendUint32(tagstart + tagsize, result);
-        AppendUint32(tagsize, result);
-        AppendKeyword(kBxyzTag, result);
-        AppendUint32(tagstart + tagsize * 2, result);
-        AppendUint32(tagsize, result);
+        JXL_QUIET_RETURN_IF_ERROR(AppendKeyword(kGxyzTag, result));
```
Comment (sboukortt) [inline lib/jxl/icc_codec.cc:205]: Was this meant to be `QUIET`?

**Comment by mo271** [comment]
No

Sami Boukortt ***@***.***> schrieb am Di., 27. Aug. 2024,
17:41:

File: lib/jxl/enc_icc_codec.cc [hunk 2]
```diff
@@ -101,7 +101,7 @@ inline Status EncodeVarInt(uint64_t value, PaddedBytes* data) {
   JXL_ENSURE(pos <= output_size);
   output[pos++] = static_cast<uint8_t>(value & 127);
 
-  data->resize(pos);
+  JXL_RETURN_IF_ERROR(data->resize(pos));
```
Comment (veluca93) [inline lib/jxl/enc_icc_codec.cc:104]: could also `return data->resize`

**Review by veluca93 — APPROVED** [review]

File: lib/jxl/enc_icc_codec.cc (hunk 2 above)
Comment (mo271) [reply lib/jxl/enc_icc_codec.cc]: done

File: lib/jxl/icc_codec.cc (hunk 1 above)
Comment (veluca93) [inline lib/jxl/icc_codec.cc:205]: still QUIET?

Comment (mo271) [reply lib/jxl/icc_codec.cc]: oops, I though Sami had fixed it..

Should be fixed now

**Review by veluca93 — APPROVED** [review]

**Pull request merged**